src/common/hooks/mutations/useUnFollow/index.ts

```diff
@@ -0,0 +1,20 @@
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+
+import { instance } from '~/api';
+
+export const useUnfollow = () => {
+  const followDelete = async ({ id }: { id: string }) => {
+    await instance.post(`/follow/delete/${id}`);
```

Comment:
method를 `delete`로 변경하고 `id`는 url이 아닌 request body로 전달해야 할 듯 합니다!

```suggestion
    await instance.delete('/follow/delete', { data: { id } });
```

Reply:
맞아요 완전 잘못만들었더라구요 지금은 수정상태이고 rebase해서 올려보겠습니다!